sdk/src/androidTest/java/com/karhoo/sdk/api/util/ServerRobot.kt

```diff
@@ -509,6 +509,8 @@ class ServerRobot {
 
         val QUOTE_LIST_EMPTY = QuoteList(
                 id = QuoteId(QUOTE_ID.quoteId),
+                status = "PROGRESSING",
```

Comment:
Are these enums?

Reply:
Documentation doesn't say.

Reply:
Got it setup in the Vehicles object and just a string there. No enums in sight and documentation around it is not clear.

Reply:
Can we try to find out? If we can use enum for quotes status or no

Reply:
Can try, not sure who to ask exactly though.